build-tools/packages/build-cli/src/library/release.ts

```diff
@@ -111,27 +117,35 @@ interface PackageTildeRange {
  * "full" corresponds to the {@link ReleaseReport} interface. It contains a lot of package metadata indexed by package
  * name.
  *
- * "simple" corresponds to the {@link PackageVersionList} interface. It contains a map of package names to versions.
+ * "simple" corresponds to the {@link PackageRange} interface. It contains a map of package names to versions.
```

Comment:
Now that there are no distinct types for each of the "formats", these comments don't make a lot of sense.